Makefile

```diff
@@ -1,15 +1,31 @@
+# LESS params
 LESS_DIR = ./static/less
-LESS_FILE = style.less
+LESS_RTL_DIR = ./static/less/rtl
+LESS_TMP_FILE = tmp.less
+
+# CSS params
 CSS_DIR = ./static/css
 CSS_FILE = style.min.css
+CSS_RTL_FILE = style-rtl.min.css
 CSS_TMP_FILE = tmp.css
 
 .PHONY: clean demo build
 
 build: clean
-	lessc $(LESS_DIR)/$(LESS_FILE) > $(CSS_DIR)/$(CSS_TMP_FILE)
+	for f in "`find $(LESS_DIR) -maxdepth 1 -type f`"; do cat $$f >> $(LESS_DIR)/$(LESS_TMP_FILE); done
+	for f in "`find $(LESS_RTL_DIR) -maxdepth 1 -type f`"; do cat $$f >> $(LESS_RTL_DIR)/$(LESS_TMP_FILE); done
+
+	# LTR
+	lessc $(LESS_DIR)/$(LESS_TMP_FILE) > $(CSS_DIR)/$(CSS_TMP_FILE)
 	uglifycss $(CSS_DIR)/$(CSS_TMP_FILE) > $(CSS_DIR)/$(CSS_FILE)
 	rm -f $(CSS_DIR)/$(CSS_TMP_FILE)
+	rm -f $(LESS_DIR)/$(LESS_TMP_FILE)
+
+	# RTL
+	lessc $(LESS_RTL_DIR)/$(LESS_TMP_FILE) > $(CSS_DIR)/$(CSS_TMP_FILE)
+	uglifycss $(CSS_DIR)/$(CSS_TMP_FILE) > $(CSS_DIR)/$(CSS_RTL_FILE)
+	rm -f $(CSS_DIR)/$(CSS_TMP_FILE)
+	rm -f $(LESS_RTL_DIR)/$(LESS_TMP_FILE)
```

Comment:
And I think we can make another separate build step just to avoid the repetition of these 4 lines.

Reply:
That's a very good idea! I'll do it when we sorted out what we want to do with files/subfolder.